internal/store/installation.go.orig

```diff
@@ -28,9 +28,9 @@ func init() {
 			"MattermostEnvRaw", "PriorityEnvRaw", "SingleTenantDatabaseConfigRaw", "ExternalDatabaseConfigRaw",
 			"Installation.CreateAt", "Installation.DeleteAt", "Installation.DeletionPendingExpiry",
 <<<<<<< HEAD
-			"APISecurityLock", "LockAcquiredBy", "LockAcquiredAt", "CRVersion", "Installation.DeletionLocked", "AllowedRanges",
+			"APISecurityLock", "LockAcquiredBy", "LockAcquiredAt", "CRVersion", "Installation.DeletionLocked", "AllowedIPRanges",
```

Comment:
Accidentally added this I think?

Reply:
@nickmisasi for the array of strings, needed an external library dependancy and in my humble opinion the benefits were low to add this extra complexity, but sure I am not strongly opinionated on this.